lib/bundler/templates/newgem/test/test_newgem.rb.tt

```diff
@@ -0,0 +1,14 @@
+$LOAD_PATH.unshift File.expand_path("../lib", __dir__)
+require "<%= config[:namespaced_path] %>"
+
+require "test/unit"
```

Comment:
`require "test-unit"` is better to ensure using test-unit gem.
(We couldn't require test-unit gem by `require "test/unit"` with old Ruby. Ruby 2.4 or later aren't old Ruby. So we can use `require "test/unit" here recently.)